transforms/no-vars.js

```diff
@@ -24,6 +24,12 @@ export default function(file, api) {
     }
     return {scopeNode, isInFor};
   };
+  const findFunctionDeclaration = (node, container) => {
```

Comment:
Why not ArrowFunctionExpressions, Generator functions, or other execution/block scopes?